common/src/app/common/types/shape_tree.cljc

```diff
@@ -240,6 +240,22 @@
                -1))]
      (sort comp ids))))
 
+(defn sort-z-index-objets
```

Comment:
If unstable-sort is faster than default sort, didn't it be useful to replace it in all calls of sort-z-index?

Or what tradeoffs does it have?

Reply:
The stable vs unstable algorithms differs in how equal elements are treated. Stable algorithm preserves relative position of equal elements, unsable does not cares about that.
In our case, this is pretty irrelevant and using unsable algorithm removes a O(n) loop from sorting.
I have not changed it on sort-z-index just because I want to touch a small portion of code, but yes, I put a note here to replace it also on sort-z-indez impl